sdk/identity/identity/review/identity.api.md

```diff
@@ -169,7 +169,7 @@ export function getDefaultAzureCredential(): TokenCredential;
 export { GetTokenOptions }
 
 // @public
-export type InteractiveBrowserAuthenticationFlow = "implicit-grant" | "auth-code";
+export type InteractiveBrowserAuthenticationFlow = "auth-code";
```

Comment:
Since it's just `auth-code` maybe we dont need this type at all? I'm assuming it's relatively new but if we GA'd with it ignore me 😄

Reply:
Good point. I'll remove it!